libs/cart/testing/src/factories/cart-address.factory.spec.ts

```diff
@@ -19,19 +19,6 @@ describe('Cart | Testing | Factories | CartAddressFactory', () => {
     expect(cartAddressFactory).toBeTruthy();
   });
 
-  describe('create', () => {
-
-    let result: DaffCartAddress;
-
-    beforeEach(() => {
-      result = cartAddressFactory.create();
-    });
-    
-    xit('should return a CartAddress with all required fields defined', () => {
-
-    });
-  });
-
   describe('createMany', () => {
```

Comment:
realistically, we should probably only test the create method in any of the factories, and only test the createMany in the CoreFactory or whatever it's called